.zuul.yaml

```diff
@@ -0,0 +1,27 @@
+---
+- project:
+    name: openstack-k8s-operators/test-operator
+    github-check:
```

Comment:
periodic?

Reply:
Yes, true. The github-check pipeline was used in order to be able to test the job and see whether zuul works correctly in the test-operator repo.